kahuna/public/js/image/service.js

```diff
@@ -25,11 +25,16 @@ imageService.factory('imageService', [function() {
     }
 
     function getStates(image) {
+        const persistReasons = image.data.persisted.reasons;
+
         return {
             cost: image.data.cost,
             hasCrops: hasExportsOfType(image, 'crop'),
             isValid: image.data.valid,
-            canDelete: image.getAction('delete').then(action => !! action)
+            canDelete: image.getAction('delete').then(action => !! action),
+            canArchive: image.data.persisted.value === false ||
```

Comment:
So are we saying you can't archive this if it's already persisted?
If so - why not? That's not true on the API level is it?